model_api/python/openvino/model_api/adapters/onnx_adapter.py

```diff
@@ -0,0 +1,207 @@
+"""
+ Copyright (c) 2023 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import sys
+from functools import partial, reduce
+
+import numpy as np
+
+from .utils import INTERPOLATION_TYPES, RESIZE_TYPES, InputTransform
+
+try:
+    import onnx
+    import onnxruntime as ort
+    from onnxruntime.tools.symbolic_shape_infer import SymbolicShapeInference
+
+    onnxrt_absent = False
+except ImportError:
+    onnxrt_absent = True
+
+from .inference_adapter import InferenceAdapter, Metadata
+from .utils import Layout, get_rt_info_from_dict, load_parameters_from_onnx
+
+
+class ONNXRuntimeAdapter(InferenceAdapter):
+    """
+    This inference adapter allows running ONNX models via ONNXRuntime.
+    The adapter has limited functionality: it supports only image models
+    generated by OpenVINO training extensions (OTX). Each onnx file
+    generated by OTX contains ModelAPI-style metadata, which is used for
+    configuring a particular model acting on top of it.
+    Also, this adapter doesn't provide asynchronous inference functionality and model reshaping.
+    Visit https://github.com/openvinotoolkit/training_extensions/ for the
+    comprehensive list of models available.
```

Comment:
Here you refer to OTX, but in readme you list
> `SSD`, `MaskRCNNModel`, `SegmentationModel`, and `ClassificationModel` wrappers.

Please, align the messages

Reply:
Updated: particular adapters are mentioned now, like in reamde